crt_portal/api/views.py

```diff
@@ -71,6 +72,15 @@ class ResponseList(generics.ListAPIView):
     permission_classes = [permissions.IsAuthenticated]
 
 
+class ResponseTitleList(generics.ListAPIView):
+    """
+    API endpoint that returns response titles.
+    """
+    queryset = ResponseTemplate.objects.all()
+    serializer_class = ResponseTitleSerializer
+    permission_classes = [permissions.IsAuthenticated]
```

Comment:
I thought this line shouldn't be necessary, due to 
```
REST_FRAMEWORK = {
    'DEFAULT_PERMISSION_CLASSES': (
        'rest_framework.permissions.IsAuthenticated',
    )
}
```

However, tests break when I remove `permission_classes = [permissions.IsAuthenticated]`.  I'm not sure what the utility of `DEFAULT_PERMISSION_CLASSES` is - does it accomplish anything?